### PR TITLE
feat(linux): keyman:// protocol handler

### PR DESCRIPTION
**Pull request by ermshiperete**
This change implements the custom keyman:// protocol handler which
will cause the browser to launch km-config when it encounters a
URL using the keyman protocol.

Related to #3271.

### CODE REVIEW

File: linux/keyman-config/tests/test_handle_install.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        mockPackagePath = '/tmp/sil_euro_latin'
+        keymanCacheDirMethod.return_value = '/tmp'
+        downloadKmpFileMethod.return_value = mockPackagePath
+
+        # Execute
+        download_and_install_package('keyman://download/keyboard/sil_euro_latin')
+
+        # Verify
+        downloadKmpFileMethod.assert_called_with(
+            KeymanComUrl + '/go/package/download/sil_euro_latin?platform=linux&tier=alpha',
```
Comment (mcdurdin) [inline linux/keyman-config/tests/test_handle_install.py:80]: This assertion will fail for beta and stable builds?

File: linux/keyman-config/tests/test_handle_install.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        mockPackagePath = '/tmp/sil_euro_latin'
+        keymanCacheDirMethod.return_value = '/tmp'
+        downloadKmpFileMethod.return_value = mockPackagePath
+
+        # Execute
+        download_and_install_package('keyman://download/keyboard/sil_euro_latin?bcp47=de')
+
+        # Verify
+        downloadKmpFileMethod.assert_called_with(
+            KeymanComUrl + '/go/package/download/sil_euro_latin?platform=linux&tier=alpha',
```
Comment (mcdurdin) [inline linux/keyman-config/tests/test_handle_install.py:100]: Ditto on assertion failure?

File: linux/keyman-config/tests/test_handle_install.py (hunk 1 above)
Comment (ermshiperete) [reply linux/keyman-config/tests/test_handle_install.py]: done

File: linux/keyman-config/tests/test_handle_install.py (hunk 2 above)
Comment (ermshiperete) [reply linux/keyman-config/tests/test_handle_install.py]: done

**Pull request merged**